griptape/tools/web_scraper/tool.py

```diff
@@ -25,7 +25,7 @@ def get_content(self, params: dict) -> ListArtifact | ErrorArtifact:
         if isinstance(page, ErrorArtifact):
             return page
         else:
-            return ListArtifact(TextLoader().text_to_artifacts(page.get("text")))
+            return ListArtifact(TextLoader().text_to_artifacts(page["text"]))
```

Comment:
this seems like it could open us up to improperly handled errors when the key "text" does not exist in order to get rid of the pyright errors. Is it a guarantee that "text" always exists?

If not, we could do something like this:
```
text = page.get("text")
if text is None:
    return ErrorArtifact("")
return ListArtifact(TextLoader().text_to_artifacts(text))
```
Funny enough, this gives me a different Pyright error about wanting Sequence instead of list.

Reply:
These keys are guaranteed to exist, yes. Here is the result of a [simple site](https://cv.bt.ht/):

```json
{
  "title": "CV",
  "author": null,
  "hostname": null,
  "date": "2023-01-01",
  "fingerprint": "81ccfd77a5f0f861",
  "id": null,
  "license": null,
  "comments": "",
  "raw_text": "Hi, I'm Brad. Resume: Fullstack Engineer @ SerpApi, 2023-Sr. Product Designer @ Donorbox, 2021-2023Web Designer @ Purism, 2019-2021Product Designer @ Benbria, 2013-2019Web Designer @ Netvatise, 2009-2013 Links: bt.htsr.ht/~bt brad@bt.ht",
  "text": "Hi, I'm Brad.\nResume:\nFullstack Engineer @ SerpApi, 2023-Sr. Product Designer @ Donorbox, 2021-2023Web Designer @ Purism, 2019-2021Product Designer @ Benbria, 2013-2019Web Designer @ Netvatise, 2009-2013\nLinks:\nbt.htsr.ht/~bt\nbrad@bt.ht",
  "language": null,
  "image": null,
  "pagetype": null,
  "source": null,
  "source-hostname": null,
  "excerpt": null,
  "categories": "",
  "tags": ""
}
```

I updated the logic though, it was previously checking for an `ErrorArtifact` from `WebLoader` which it does not return.